### ISSUE TITLE
sendmail.send_message_via_ses doesn't exist

### ISSUE THREAD

**Issue by viking** [mitro-co/mitro#111]
See https://github.com/mitro-co/mitro/blob/c51aa3503e0b32377c0fd9baad274eb586b19509/emailer/auth/email_queue.py#L98.

This function is not provided by the sendmail module.

**Comment by alcazoid** [comment]
Yes, I have this too, it prevents me from getting verification code. I'm running debian stable.
